Makefile.uk

```diff
@@ -0,0 +1,143 @@
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_aux.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_close.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_codec.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_color.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_compress.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_dir.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_dirinfo.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_dirread.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_dirwrite.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_dumpmode.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_error.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_extension.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_fax3.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_fax3sm.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_flush.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_getimage.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_jbig.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_jpeg.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_jpeg_12.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_luv.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_lzma.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_lzw.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_next.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_ojpeg.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_open.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_packbits.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_pixarlog.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_predict.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_print.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_read.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_strip.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_swab.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_thunder.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_tile.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_version.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_warning.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_write.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_zip.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/libtiff/tif_unix.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/contrib/addtiffo/tif_overview.c
+LIBTIFF_SRCS-y += $(LIBTIFF_EXTRACTED)/contrib/addtiffo/tif_ovrcache.c
+
+################################################################################
+# TIFF prepare
+################################################################################
+# Run ./configure
+$(LIBTIFF_EXTRACTED)/config.status: $(LIBTIFF_BUILD)/.origin
+	$(call verbose_cmd,CONFIG,libtiff: $(notdir $@), \
+		cd $(LIBTIFF_EXTRACTED) && ./configure)
+
+LIBTIFF_PREPARED_DEPS = \
+	$(LIBTIFF_EXTRACTED)/config.status \
+
+$(LIBTIFF_BUILD)/.prepared: $(LIBTIFF_PREPARED_DEPS)
```

Comment:
This step needs to be removed.  What is required from the configure step are the `libtiff/tiffconf.h` and `libtiff/tif_config.h` files which need to be explicitly created and placed within a top-level `include/` directory of this repo.  The directory and corresponding header file should be referenced via `LIBTIFF_COMMON_INCLUDES`.  It's options need be checked and must correspond with what is compatible with Unikraft.